### PR TITLE
Adds a sound effect to becoming a Revolutionary like all other antagonists.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Adds a sound effect to becoming a Revolutionary like all other antagonists.

## Why It's Good For The Game

Sound effects are good, and despite being one of our most veteran gamemodes and antagonists, Revolutionaries lacked a sound.

https://user-images.githubusercontent.com/4081722/161388001-1aeca3c6-fe89-4614-b5dc-82938f690a03.mp4

https://user-images.githubusercontent.com/4081722/161388105-b7b32cff-a40f-4ed8-9cbe-d5b43286f0e0.mp4

Copyright shouldn't be an issue, The Internationale is the Internationale.
## Changelog

:cl:
soundadd: Adds a sound effect to becoming a Revolutionary like all other antagonists.
/:cl:

### CODE REVIEW

**Comment by wesoda25** [comment]
Imagine hearing this knowing you are about to witness and commit countless acts of barbarism... excellent.

As an aside, why not have this play when revs win as well?

**Comment by Iamgoofball** [comment]
we should play the full thing when they win

**Comment by Ghommie** [comment]
> we should play the full thing when they win

Yay, let's have players download 6 more megabytes of cache resources to keep their ears too busy to hear what's going on in-game (and irl) for over 4 minutes straight with this majestic anthem whenever they win as revolutionaries without opt-out short of the Stop Sounds verb not everyone knows about. You should be given extra GBPs if this gets merged as is.

I'll make myself clearer here if the sarcastic overtone of the previous paragraph wasn't catched on by someone: This is a good PR and The Internationale is a great choice - also public domain - for revs, but I doubt making people listen to all of it on victory is likewise a good idea, mostly because it'll cover up other sounds which is something generally undesiderable for the player. Also, that file alone will increase the size of the sounds folder by about 7%.

**Comment by Iamgoofball** [comment]
I will shrink the file

File: code/modules/antagonists/revolution/revolution.dm [hunk 1]
```diff
@@ -14,6 +14,8 @@
 	var/datum/team/revolution/rev_team
 	///when this antagonist is being de-antagged, this is why
 	var/deconversion_reason
+	/// What sound should we play for becoming this brand of revolutionary?
+	var/internationale = 'sound/ambience/antag/viva_short.ogg'
```
Comment (Mothblocks) [inline code/modules/antagonists/revolution/revolution.dm:18]: Rename this variable to `gain_sound` or similar to signify its actual meaning.

Comment (Iamgoofball) [reply code/modules/antagonists/revolution/revolution.dm:18]: yeet

**Comment by Iamgoofball** [comment]
@Ghommie i made it tiny w/o sacrificing the audio quality

**Comment by Shadowflame909** [comment]
Jolly and cheerful compared to all the mass murdering they will be doing. 

But I guess when theres nothing left to lose, all you have to do is cheer.

**Comment by ProfessorSniff** [comment]
> Copyright shouldn't be an issue, The Internationale is the Internationale.

Did you get this from Wikipedia? It sounds identical, and they have it marked as non-free:
image
But their only source for the audio is [a 10 year old youtube video that *probably* didn't record it](https://www.youtube.com/watch?v=FGqMMpMkKXs) so we really don't know where this recording of the song came from, so it probably isn't copyright friendly?

if this recording is copyrighted, that would be rather ironic

**Comment by Iamgoofball** [comment]
i got it from the youtube video so that means I got it from the true source 😎

**Comment by ProfessorSniff** [comment]
that doesnt improve the copyright situation :skull:

**Comment by Iamgoofball** [comment]
It does not, no. I'll talk to the sound people.

**Comment by Ghommie** [comment]
Uh, yea, I forgot it's the original sheet music that's public domain. Of course many orchestras would put a copyright to the results of their instrumentation anyway. Oh, the devil is in the details.

**Comment by Mothblocks** [comment]
DNM until you figure out the copyright situation

**Comment by BlueMemesauce** [comment]
@Iamgoofball @Mothblocks someone added a Creative Commons version to Wikipedia. Maybe you can reopen this and use that file.

https://en.wikipedia.org/wiki/File:The_Internationale_Bolshoi_version.flac

**Comment by BlueMemesauce** [comment]
Alternatively you could use this completely copright free version https://en.wikipedia.org/wiki/File:Internationale_orchestral_arrangement.ogg

**Comment by Iamgoofball** [comment]
yeah hit me up post freeze

**Comment by DrDiasyl** [comment]
@Iamgoofball no Internationale?

**Pull request closed without merging**